### ISSUE TITLE
Can't import script from folder - No module named 'data'

### ISSUE THREAD

**Issue by yinqihuang** [julrog/nn_vis#26]
**Code:**
` from data.mnist_data_handler import split_mnist_data`

**Error message:**
`C:\Users\Yinqi Huang\nn_vis>python examples/process_mnist_model.py
Traceback (most recent call last):
  File "C:\Users\Yinqi Huang\nn_vis\examples\process_mnist_model.py", line 4, in <module>
    from data.mnist_data_handler import split_mnist_data
ModuleNotFoundError: No module named 'data'`

**Comment by julrog** [comment]
I have only tested it on windows with pycharm, where everything works fine (for me), but I think in your case maybe adding the __init__.py files in the directories will fix this. Either you try his, or wait until I will fix it after reproducing your error.

**Comment by julrog** [comment]
I know what is causing the issue. Until i fix it in a better way, for now you can simply add following at the top of the `.py` file you want to run:
```
import os,sys,inspect
currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
parentdir = os.path.dirname(currentdir)
sys.path.insert(0,parentdir)

os.chdir('../')
```
from https://stackoverflow.com/questions/714063/importing-modules-from-parent-folder

**Comment by julrog** [comment]
Fixed in commit 17403ab33cf215026d0b011d36ef612f1c08055f .
